nservicebus/messaging/databus/custom-databus_content_core_[6.0,7.0].partial.md

```diff
@@ -0,0 +1,33 @@
+It is possible to create a custom DataBus implementation. This is done by making use of the [Features](/nservicebus/pipeline/features.md) extension.
+
+
+## Implement the `IDataBus` interface. 
+
+This new class will provide the custom implementations for the `Get` and `Put` methods for the DataBus. 
+
+snippet: CustomDataBus
+
+This new implementation needs to be now registered as a new feature. 
+
+
+## Define a feature 
+
+Define a [new feature](/nservicebus/pipeline/features.md) for the custom databus by creating a class that inherits from `Feature`. In the Setup method, register the custom DataBus implementation class.
```

Comment:
I would simplify this to just

> Define a new feature that registers the custom DataBus implementation class. 

Try not to describe the details of the snippet. That way, if the snippet changes, the text remains accurate.